pyuvdata/uvdata/ms.py

```diff
@@ -119,6 +119,63 @@ def _write_ms_antenna(self, filepath):
             antenna_table.putcol("DISH_DIAMETER", ant_diam_table)
         antenna_table.done()
 
+    def _write_ms_feed(self, filepath):
+        """
+        Write out the feed information into a CASA table.
+
+        Parameters
+        ----------
+        filepath : str
+            path to MS (without FEED suffix)
+
+        """
+        if not casa_present:  # pragma: no cover
+            raise ImportError(no_casa_message) from casa_error
+
+        feed_table = tables.table(filepath + "::FEED", ack=False, readonly=False)
+
+        nfeeds_table = np.max(self.antenna_numbers) + 1
+        feed_table.addrows(nfeeds_table)
+
+        antenna_id_table = np.arange(nfeeds_table, dtype=np.int32)
+        feed_table.putcol("ANTENNA_ID", antenna_id_table)
+
+        beam_id_table = np.ones(nfeeds_table, dtype=np.int32)
+        feed_table.putcol("BEAM_ID", beam_id_table)
+
+        beam_offset_table = np.zeros((nfeeds_table, 2, 2), dtype=np.float64)
```

Comment:
I need to understand what this means. I'm worried about just setting it to zero for pointing arrays like MWA.

Reply:
@kartographer do you understand what this means?

Reply:
I believe this column is used for handling pointing offsets for an individual primary beam, i.e. when the primary beam isn't pointed at the phase center of the array. But because we set BEAM_ID=-1 above (and we don't have a BEAM table _yet_), none of this gets used at the moment. At some point further down the road, one ought to be able to fill some of that in with a `UVBeam` object.